io.sloeber.core/src/io/sloeber/core/tools/uploaders/arduinoUploader.java

```diff
@@ -111,20 +104,16 @@ public boolean uploadUsingPreferences(IFile hexFile, boolean usingProgrammer, IP
 	 */
 	@SuppressWarnings("nls")
 	private String getPasswordFromCode() {
-		return IndexHelper.findParameterInFunction(this.myProject, "setup", "ArduinoOTA.setPassword",
+		String parameter = IndexHelper.findParameterInFunction(this.myProject, "setup", "ArduinoOTA.setPassword",
```

Comment:
It has a lot of shortcomings (doesn't always work, can't change password, intransparent what's going on/why it's not working) therefore I believe offering an alternative would be nice